boards/silabs/starter_kits/slstk3701a/slstk3701a.dts

```diff
@@ -92,6 +92,12 @@
 	status = "okay";
 };
 
+&i2c2 {
+	pinctrl-0 = <&i2c2_default>;
+	pinctrl-names = "default";
+	status = "okay";
```

Comment:
I am not super familiar with [devices that must enabled by default][1], but I believe we have to keep i2c `"disabled"`.

[1]: https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#general-recommendations

Reply:
That's said, `i2c0` and `i2c1` are enable. So, I believe this patch is correct.